homeassistant/components/smarttub/controller.py

```diff
@@ -86,15 +86,17 @@ async def async_update_data(self):
         return data
 
     async def _get_spa_data(self, spa):
-        status, pumps, lights = await asyncio.gather(
+        status, pumps, lights, reminders = await asyncio.gather(
             spa.get_status(),
             spa.get_pumps(),
             spa.get_lights(),
+            spa.get_reminders(),
         )
         return {
             "status": status,
             "pumps": {pump.id: pump for pump in pumps},
             "lights": {light.zone: light for light in lights},
+            ATTR_REMINDERS: {reminder.id: reminder for reminder in reminders},
```

Comment:
I updated the rest of these to use constants in https://github.com/home-assistant/core/pull/47584